internal/utils/indexes.go

```diff
@@ -19,10 +19,11 @@ import (
 	"path/filepath"
 
 	"github.com/falcosecurity/falcoctl/pkg/index"
+	"github.com/falcosecurity/falcoctl/pkg/index/config"
 )
 
 // Indexes returns the merge of all configured indexes.
-func Indexes(indexConfig *index.Config, path string) (*index.MergedIndexes, error) {
+func Indexes(indexConfig *config.Config, path string) (*index.MergedIndexes, error) {
```

Comment:
Seems to be dead code, It would be nice to remove it. I think it's a leftover from previous refactors.